docs/designers-developers/developers/tutorials/notices/README.md

```diff
@@ -0,0 +1,77 @@
+# Displaying Notices from Your Plugin or Theme
+
+Notices are informational UI displayed near the top of admin pages. WordPress, themes, and plugins all use notices to indicate the result of an action, or to draw the user's attention to necessary information.
```

Comment:
```suggestion
Notices are informational UI displayed near the top of admin pages. WordPress core, themes, and plugins all use notices to indicate the result of an action, or to draw the user's attention to necessary information.
```

Reply:
Is "WordPress core" official verbiage? I feel like it might not be the technically correct phrasing.

Reply:
We've been using it in various tutorials but I'm fine not worrying about it here.